### PR TITLE
Ckanext validation: remove schema upload UI component

### PR DESCRIPTION
**Pull request by KatiRG**
## What this PR accomplishes
Removes the UI component for uploading own schema in the resource edit page

## Issue addressed
DATA-1346

## What needs review
- confirm that the schema upload component does not appear in the resource Edit page
- ckanext-validation still able to validate a new resource. Note:
    - don't test on existing resources, that's not implemented yet
    - recommended to test on a valid file. Uploading an invalid file will cause the 504 error (known bug)

### CODE REVIEW

File: ckanext/ontario_theme/plugin.py [hunk 1]
```diff
@@ -177,6 +177,80 @@ def resource_display_name(resource_dict):
 
 ckan.lib.helpers.resource_display_name = resource_display_name
 
+# def get_datastore_info(resource_id):
```
Comment (KatiRG) [inline ckanext/ontario_theme/plugin.py:180]: ignore this file

File: ckanext/ontario_theme/schemas/external/ontario_theme_dataset.json [hunk 2]
```diff
@@ -882,5 +883,27 @@
       "validators": "ignore_missing unicode package_version_validator",
       "classes": ["form-group","col-md-12"]
     }
+    ,
+    {
+      "field_name": "schema",
+      "label": "Schema",
+      "preset": "resource_schema"
```
Comment (Alexandra-Green) [inline ckanext/ontario_theme/schemas/external/ontario_theme_dataset.json:890]:
```suggestion
      "preset": "hidden_in_form"
```
If you use the "hidden_in_form" preset, you can remove `resource_schema.html` altogether and achieve the same results

File: ckanext/ontario_theme/templates/internal/scheming/form_snippets/resource_schema.html [hunk 3]
```diff
@@ -0,0 +1 @@
+{% import 'macros/form.html' as form %}
```
Comment (Alexandra-Green) [inline ckanext/ontario_theme/templates/internal/scheming/form_snippets/resource_schema.html:1]: You can remove this file altogether with the `"hidden_in_form"` preset (see comment above)

File: ckanext/ontario_theme/templates/internal/scheming/package/resource_read.html [hunk 4]
```diff
@@ -151,6 +151,8 @@ <h2 class="ontario-h4">
           </div>
         {% endif %}
         {% if res.datastore_active %}
+          {% set info = h.ontario_theme_get_datastore_info(res.id) %}
+          <p>{{info}}</p>
```
Comment (Alexandra-Green) [inline ckanext/ontario_theme/templates/internal/scheming/package/resource_read.html:155]: Are these lines still required?

**Review by Alexandra-Green — CHANGES_REQUESTED** [review]

**Review by Alexandra-Green — APPROVED** [review]

**Pull request merged**